base.json5

```diff
@@ -3,32 +3,36 @@
   description: "strg base config",
   extends: [
     // https://docs.renovatebot.com/presets-config/
-    "config:best-practices",
```

Comment:
I did split this up to remove ":ignoreModulesAndTests". thats why there are additional presets now.